AuthorizationService/AuthorizationService/Program.cs

```diff
@@ -16,22 +19,34 @@ public static void Main(string[] args)
             #region Services
             var connectionString = builder.Configuration.GetConnectionString("sqlConnection");
 
-            builder.Services.AddMediatR(Assembly.GetExecutingAssembly());
+            builder.Services.AddDbContext<RepositoryContext>(
+                options =>
+                options.UseSqlServer(connectionString,
+                x => x.MigrationsAssembly("AuthorizationService")));
+            builder.Services.AddScoped<IRepositoryManager, RepositoryManager>();
+
+            builder.Services.AddIdentity<Account, IdentityRole>(config =>
+            {
+                config.Password.RequiredLength = 4;
+                config.Password.RequireNonAlphanumeric = false;
+                config.Password.RequireLowercase = false;
+                config.Password.RequireUppercase = false;
+            })
```

Comment:
Configure this using IConfigureOptions and store configuration inside appsettings.json